app/models/validations/shared_validations.rb

```diff
@@ -68,4 +68,15 @@ def inactive_status(date, resource)
 
     { scope: status, date: date&.to_formatted_s(:govuk_date), deactivation_date: closest_reactivation&.deactivation_date&.to_formatted_s(:govuk_date) }
   end
+
+  def validate_valid_radio_option(record)
```

Comment:
Unfortunately we need to resolve these issues before we can move forward with this PR: https://communities-govuk.slack.com/archives/C01QGR78H9S/p1673357370076599

Reply:
The good news is that they are straight forward data migrations.